src/calculateRentalCost.js

```diff
@@ -27,8 +27,20 @@
  * @return {number}
  */
 
+const DAILY_PRICE = 40;
+const MIN_DISCOUNT = 20;
+const MAX_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  let totalPrice = DAILY_PRICE * days;
+
+  if (days >= 7) {
+    totalPrice -= MAX_DISCOUNT;
+  } else if (days >= 3) {
+    totalPrice -= MIN_DISCOUNT;
+  }
```

Comment:
You can just use early return instead of `else if `nesting. Try to avoid else/if. It makes your code clean
```suggestion
  if (days >= 7) {
   return totalPrice - MAX_DISCOUNT;
  } 
  if (days >= 3) {
    return totalPrice - MIN_DISCOUNT;
  }
```